assets/javascripts/test_result.js

```diff
@@ -586,6 +586,20 @@ function setCurrentPreviewFromStepLinkIfPossible(stepLink) {
   }
 }
 
+function githashToLink(value) {
+  var hashes_regex = /(\b[0-9a-f]{9}\b)/gms;
+  var githashes = value.match(hashes_regex);
+  var repo = 'https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/';
+  var stat_regex = /(\<a.+?)(?=(\<a))/gms;
```

Comment:
That would be our normal coding style (in JavaScript):
```suggestion
  var statRegex = /(\<a.+?)(?=(\<a))/gms;
```

The same counts for the many other occurrences but it is also not *that* important.

Reply:
this change is ready for review